### ISSUE TITLE
Add rounded flags as well

### ISSUE THREAD

**Issue by vanlooverenkoen** [LunaGao/flag_flutter#58]
Something like this:

![image](https://user-images.githubusercontent.com/21172855/128020805-b1db2549-a456-43a7-812d-bf40063f8389.png)

**Comment by Rohithgilla12** [comment]
@vanlooverenkoen You can wrap it around with `ClipRRect`

Something like this
```
ClipRRect(
    borderRadius:
        BorderRadius.circular(AppDimensions.mainSpace),
    child: Flag.fromCode(
      FlagsCode.US,
      height: 32,
      width: 32,
      fit: BoxFit.fill,
    ),
  ),
  
  ```

**Comment by vanlooverenkoen** [comment]
Yes but that would not work for every flag. It will cut off parts of the flag that are important for some countries. (custom rounded icons would be the better solution in my opinion)

**Comment by LunaGao** [comment]
fixed in v6.0.0
